apiserver/getnodebootstrapdata.go

```diff
@@ -17,60 +17,39 @@ const (
 	RoutePathNodeBootstrapData string = "/getnodebootstrapdata"
 )
 
-func handleError(err error) Result {
-	log.Println(err.Error())
-	return Result{"", err}
-}
-
 // GetNodeBootstrapConfig endpoint for getting node bootstrapping data.
 func (api *APIServer) GetNodeBootstrapData(w http.ResponseWriter, r *http.Request) {
 	ctx := r.Context()
 	ctx, cancel := context.WithTimeout(ctx, 30*time.Second)
 	defer cancel()
 
-	processResult := make(chan Result)
-	go func() {
```

Comment:
doing this in a goroutine avoided our panic handler which swallows a lot of useful info.